presto-main/src/main/java/com/facebook/presto/execution/QueryStats.java

```diff
@@ -452,4 +460,10 @@ public OptionalDouble getProgressPercentage()
         }
         return OptionalDouble.of(min(100, (completedDrivers * 100.0) / totalDrivers));
     }
+
+    @JsonProperty
```

Comment:
We keep getters in the same order as the member variables, please put this after `getOutputPositions`